### DISCUSSION TITLE
Is there a way to remove the notifications icon from the taskbar?

### DISCUSSION THREAD

**Discussion by cheonmux** [valinet/ExplorerPatcher#2947] [Q&A] [answered]
hello. I'm using windows 10 taskbar with EP on windows 11 23H2.

But I want to remove this unused notification icon, but I can't find it in the options in EP.

![2024-03-07 20 12 25](https://github.com/valinet/ExplorerPatcher/assets/15154697/f9fb5825-4ba2-4813-9372-f96881b2c286)

If anyone knows how to remove this icon, could you please let me know?

Thanks.

**Comment by pyrates999** [comment, marked as answer]
Try this: https://www.makeuseof.com/disable-notification-area-windows/

But be aware that is used by any application that sends you a notification, including if there is windows updates available, and if windows updates were successfully installed or not.  It's also how you get to the action center.  So it does serve a purpose.

**Reply by cheonmux** [reply]
It`s works perfectly! Thank you!

**Comment by basi01** [comment]
It's just too big. I'd prefer to have a regular tray icon to see past notifications.

**Reply by pyrates999** [reply]
Please create a new discussion as this one has already been answered.
